src/google.py

```diff
@@ -59,14 +65,55 @@ def login():
         
     except ValueError as ex:
         return  jsonify({
-            'error': " invalid id_token"
+            'error': "invalid id_token"
         }), HTTP_400_BAD_REQUEST
     
     
+    issuer = True if os.environ.get('TOKEN_ISSUER') == google_response.get('iss') else False
+    valid_client = True if os.environ.get('CLIENT_ID') == google_response.get('aud') else False
+    token_exp = True if time.time() > int(google_response.get('exp')) else False
```

Comment:
We don't necessarily need these three lines. See comment below on how to fix this in the rest of the code.